app/src/main/res/layout/fragment_movie_detail.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".fragments.MovieDetailFragment"
+    android:background="@color/background_teal"
+    >
+
+    <TextView
+        android:id="@+id/tv_title"
+        android:layout_width="336dp"
```

Comment:
try to stop using fixed sizes. Use relative positioning utilizing wrap_content/match_parent + margins/paddings